lib/compress/zstd_compress.c

```diff
@@ -1854,6 +1854,20 @@ ZSTD_reduceTable_internal (U32* const table, U32 const size, U32 const reducerVa
 
     for (rowNb=0 ; rowNb < nbRows ; rowNb++) {
         int column;
+#ifdef __aarch64__
```

Comment:
Do you see gains from this? This gets called once every few GB after compressing 3.5 GB. I suspect that this isn't worth the complexity, unless you have a use case that proves otherwise.

Reply:
Does this actually gain anything? I wouldn't expect it to, but your use case may be unique.

Reply:
For the modification, I tested Level1~17 with lzbench (gcc9.2.0), and only several levels were impoved.